docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,403 @@
+
+For reproducibility, governance, auditing and sharing of algorithmic systems it is essential to have a
+reporting standard so that information about an algorithmic system can be shared. This reporting standard
+describes how information about the different phases of an algorithm's life cycle can be reported.
+It contains, among other things, descriptive information combined with information about the technical
+tests and assessments applied.
+
+## Disclaimer
+
+The TAD Reporting Standard is work in progress. This means that the current standard is probably suboptimal
+and will change significantly in future versions.
+
+## Introduction
+
+Inspired by [Model Cards for Model Reporting](https://arxiv.org/abs/1810.03993)
+and [Papers with Code Model Index](https://github.com/paperswithcode/model-index) this standard almost
+[^1] [^2] [^3] [^4] extends the [Hugging Face model card metadata specification](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1)
+to allow for:
+
+1. More finegrained information on performance metrics, by extending the `metrics_field` from the Hugging
+Face metadata specification.
+2. Capturing additional *measurements* on fairness and bias, which can be partitioned into bar plot like
+measurements (such as mean absolute SHAP values) and graph plot like measurements (such as partial dependence). This is achieved
+by defining an new field `measurements`.
+3. Capturing *assessments* (such as [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)
+and [ALTAI](https://digital-strategy.ec.europa.eu/en/library/assessment-list-trustworthy-artificial-intelligence-altai-self-assessment)).
+This is achieved by defining a new field `assessments`.
+
+Following Hugging Face, this proposed standard will be written in yaml.
+
+This standard does not contain all fields present in the Hugging Face metadata specification. The fields that
+are optional in the HuggingFace specification and are specific to the HugginFace interface are ommited.
+
+Another difference is that we devide our implementation into three seperate parts.
+
+1. `system_card`, containing information about a group of ML-models which accomplish a specific task.
+2. `model_card`, containing information about a specific ML-model.
+3. `assessment_card`, containing information about a regulatory assessment.
+
+!!! note "Include statements"
+
+    These `model_card`s and  `assessment_card`s  can be included verbatim into a `system_card`,
+    or referenced with an `!include` statement, allowing for minimal cards to be compact and in a
+    single file, but for extensive cards to be split up for readability and maintainability.
+    Our standard allows for the `!include` to be used anywhere.
+
+## Intended usage
+
+The reporting standard defined in this document is intented to be given as a preamble in a README, similar to
+the Hugging Face metadata.
+
```

Comment:
I want to propose to remove this section for now. In a new version we can add examples of how it can be used.
```suggestion
```